tests/puppeteer/pages/BO/advancedParameters/webservice/add.js

```diff
@@ -0,0 +1,39 @@
+require('module-alias/register');
+const BOBasePage = require('@pages/BO/BObasePage');
+
+module.exports = class AddWebserviceKey extends BOBasePage {
+  constructor(page) {
+    super(page);
+
+    this.pageTitleCreate = 'Webservice •';
+    this.pageTitleEdit = 'Webservice •';
+
+    // Selectors
+    this.webserviceKeyInput = '#webservice_key_key';
+    this.generateButton = 'button.js-generator-btn';
+    this.keyDescriptionTextarea = '#webservice_key_description';
+    this.statusSwitchLabel = 'label[for="webservice_key_status_%ID"]';
+    this.saveButton = 'div.card-footer button';
+  }
+
+  /*
+  Methods
+   */
+
+  /**
+   * Fill form for add/edit webservice key
+   * @param webserviceData
+   * @param toGenerate
+   * @return {Promise<textContent>}
+   */
+  async createEditWebservice(webserviceData, toGenerate = true) {
```

Comment:
```suggestion
  async createEditWebservice(webserviceData) {
```
The parameter toGenerate is not used

Reply:
Because it's true by default in the current method and I used `toGenerate = true` in the test.

Reply:
the parameter is used in the function but never in the context, is it really useful to add it now?

Reply:
No, it's not useful now.
I will add the condition in the next test when I use the false value :) 
Thank you!